packages/core/src/routes/hook.ts

```diff
@@ -169,6 +172,11 @@ export default function hookRoutes<T extends ManagementApiRouter>(
       response: Hooks.guard,
       status: [201, 400],
     }),
+    koaReportSubscriptionUpdates({
+      key: 'hooksLimit',
+      quota,
+      methods: ['POST'],
```

Comment:
Can we read the methods from request context?

Reply:
Since the `koaReportSubscriptionUpdates()` is added to each API, there is no need to get the methods from request context for now since it brings additional complexity?

Reply:
And we specify the `methods` to let the code know that we want to report subscription updates when the request is with such HTTP methods. Getting the method directly from request context is not right in such scenario.